src/Main.java

```diff
@@ -0,0 +1,183 @@
+
+public class Main {
+
+
+    public static void main(String[] args) {
+
+        Farfetchd peter = new Farfetchd("Peter","","","",29,39);
+
+        peter.attack();
+        peter.makeNoise();
+
+        Unown womble = new Unown("Womble","","","",12,36,'G');
+
+        womble.healSelf();
+        womble.healSelf();
+        womble.attack();
+        womble.makeNoise();
+
+        Charjabug scrumple = new Charjabug("Scrumple","","","",8,15);
+
+        scrumple.attack();
+        scrumple.makeNoise();
+
+        Toxtricity rose = new Toxtricity("rose","","","",4,11,true);
+
+        rose.attack();
+        rose.makeNoise();
+    }
+
+}
+
+abstract class Pokémon{
```

Comment:
Je mag de klassen in een aparte file zetten, hierdoor blijft het overzichtelijk en is ziet je main er clean uit!